src/popup/pages/Assets/AssetDetailsInfo.vue

```diff
@@ -162,6 +162,21 @@
           :label="$t('pages.token-details.atl-change')"
           :text="formatCurrency(assetData.atl)"
         />
+
+        <DetailsRow
+          v-if="isAssetCoin"
+          class="link"
+          :label="$t('pages.token-details.more')"
+        >
+          <template #text>
+            <LinkButton
+              :to="assetCoinGeckoLink"
+            >
+              {{ displayCoinGeckoLink }}
+              <ExternalLink />
```

Comment:
Please use the `#icon` slot provided by the `LinkButton` component.
Also maybe the `LinkButton` should have `is-external` bool prop so the icon would be automatically added.